webhook/resources/volume/mutator.go

```diff
@@ -68,5 +206,17 @@ func mutateVolume(newObj runtime.Object) (admission.PatchOps, error) {
 		}
 	}
 
```

Comment:
Shouldn't we perform the check as lines 80-169 because the user probably changing the Volume CR.
In this case, the mutating webhook of the _Update_ operation would be called, right?

Reply:
Ideally, we should check as line 80-169 in `Update`, but there are some exclusive validations between `Update` operates. As [1], the next step is to find out more common and general checks and move them to the webhook.

Reply:
I see, thank you for the effort.